sdk/core/azure-core/azure/core/tracing/ext/opencensus.py

```diff
@@ -33,32 +51,57 @@ def __init__(self, span=None, name="parent_span"):
             span = tracer.span(name=name)
 
         self.tracer = tracer
-        self.span_instance = span
+        self._span_instance = span
+
+    @property
+    def span_instance(self):
+        # type: () -> Span
+        """
+        :return: The openencensus span that is being wrapped.
+        """
+        return self._span_instance
```

Comment:
I'm not sure I would bother with a property, just have methods internally use `self._span_instance` (i.e. this does not seem like it needs to be publicly exposed at all)

Reply:
So I use it here when I set the context. I set the context in not only our sdk context but I also make sure that the wrapper class has the span instance as the current span:
https://github.com/SuyogSoti/azure-sdk-for-python/blob/distributed_tracing/sdk/core/azure-core/azure/core/trace/base.py#L13-L17

Reply:
Ok, if it used externally than the property makes sense